jscomp/core/lam.ml

```diff
@@ -31,12 +31,15 @@ type apply_status =
 
 
 module Types = struct
-  type switch =
+  type switch_names = {consts: string array; blocks: string array}
+
```

Comment:
what do you think of the trade off using `string array` vs `string list`

Reply:
we need to do look up by index, so this choice means converting to array early on rather than on each use later on